### ISSUE TITLE
more tiny and less expensive devices support e.g. ESP8266

### ISSUE THREAD

**Issue by JGrotex** [TIBCOSoftware/flogo#144]
searching for Flogo supported Devices, more tiny and less expensive e.g. up to 6 € in best case with WLAN.
Any recommendations here ?

**Comment by mellistibco** [comment]
Yes, @jvanderl has built a number of device profiles. We'll get them added to the flogo.io showcase.

**Comment by peddamat** [comment]
Hello, any update on this?

**Comment by thanhngvpt** [comment]
hi all. What about this? I really care about working with ESP32 or ESP8266

**Comment by ch3ckmat3** [comment]
So no update on this? 

This can be huge! What can be done is to have a simple cross compiler which can compile the Go output to target these processors.
--- OR --- 
A language compiler/translator, to translate Go to C++ supported by Arduino family.

Any thoughts?

**Comment by frankcohen** [comment]
Did Flogo get ESP32 support? -Frank

**Comment by ch3ckmat3** [comment]
hello @flogo! where did you go on the ESP family?

An RPi Zero can run Node-Red, and flogo is the best bet to enable ESP32 the similar functionality. Edge computing will have huge bump if this is implemented properly.
